hash/megadriv.xml

```diff
@@ -26720,7 +26761,8 @@ Black screen, writes in ROM area then jumps to PC=1a2000 (encryption?)
 	</software>
 
 	<software name="sonic2p11" cloneof="sonic2">
-		<description>Sonic The Hedgehog 2 (World, final prototype 19921102)</description>
+		<!--	Dated 19921102	-->
+		<description>Sonic The Hedgehog 2 (final prototype)</description>
```

Comment:
Why remove the date from the description?  Did it really hurt to have it there?

Reply:
Didn't think it was relevant to have it in the description since it's the same as final but guess I'll just put it back.